server/services/address.service.js

```diff
@@ -96,3 +96,12 @@ const _convertPostcodeToUpperCase = address => {
 
   address.AddressLine += postcode
 }
+
+const _getCertificate = () => {
+  // Check if the address lookup certificate is a file or a base64 string. If a string convert it back to binary
```

Comment:
Only a minor point but this comment, although useful, should really be a method doc comment,


i.e. 

/**
 * One of these....
 */

As a side note, Inline comments ( // .... ) can generally be avoided by refactoring using sensible helper method names, as you have done in this PR. Inline comments should only be used where it is not clear *why* a bit of code is needed, it shouldn't be used to describe *what* the code is doing, the code should be self-describing.

Reply:
I've updated the comment